buildchain/buildchain/salt_tree.py

```diff
@@ -209,7 +209,7 @@ def task__deploy_salt_tree() -> Iterator[types.TaskDict]:
     Path('salt/_modules/containerd.py'),
     Path('salt/_modules/cri.py'),
     Path('salt/_modules/docker_registry.py'),
-    Path('salt/_modules/kubernetes.py'),
+    Path('salt/_modules/kubernetesmod.py'),
```

Comment:
Also, not very fond of `kubernetesmod`, maybe `k8s_utils` or sth like that would be better

Reply:
Ok, I heard the reason for behind this name so let's keep it.